src/rewrite.rs

```diff
@@ -0,0 +1,19 @@
+
+
+
+pub trait Rewrite {
+    fn rewrite(&self, width: usize, offset: usize) -> Option<(String, Option<Comment>)>;
+}
+
+pub enum Comment {
+    Simple(String),
+}
```

Comment:
At least adding a `DocComment` and `InnerDocComment`. We need to know it if we handle any reformating of the comments.